spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/SparkMicroBatchStream.java

```diff
@@ -200,25 +210,43 @@ private List<FileScanTask> planFiles(StreamingOffset startOffset, StreamingOffse
 
     StreamingOffset currentOffset = null;
 
+    // [(startOffset : startFileIndex), (endOffset : endFileIndex) )
```

Comment:
the last ) should be ]?

Reply:
added `)` in place of `]` to show the range endOffset is not inclusive.